packages/fxa-auth-server/scripts/stripe-products-and-plans-to-firestore-documents/stripe-products-and-plans-converter.ts

```diff
@@ -378,16 +378,16 @@ export class StripeProductsAndPlansConverter {
               ...productConfig.locales,
               ...(await this.stripePlanLocalesToProductConfigLocales(plan)),
             };
-          } catch (planLangError) {
-            if (planLangError.message !== PLAN_EN_LANG_ERROR) {
+          } catch (err) {
+            if (err.message === PLAN_EN_LANG_ERROR) {
```

Comment:
@biancadanforth Thank you so much for reminding me to write a test for the new try-catch.  😅   It took me a while to realize why my test wasn't working..l